casr/tests/tests.rs

```diff
@@ -2425,7 +2425,7 @@ fn test_casr_cluster_c() {
         .parse::<u32>()
         .unwrap();
 
-    assert_eq!(before_cnt, 11, "Before count mismatch.");
+    assert_eq!(before_cnt, 12, "Before count mismatch.");
```

Comment:
The proposed algorithm must not affect the already existing clustering algorithm test results.